test/SmokeTests/OOProcSmokeTests/DotNetIsolated/FaultyOrchestrators.cs

```diff
@@ -0,0 +1,155 @@
+{
+    public static class FaultyOrchestrators
+    {
+        [Function(nameof(OOMOrchestrator))]
+        public static Task OOMOrchestrator(
+            [OrchestrationTrigger] TaskOrchestrationContext context)
+        {
+            // this orchestrator is not deterministic, on purpose.
+            // we use the non-determinism to force an OOM exception on only the first replay
+            
+            // check if a file named "replayEvidence" exists in source code directory, create it if it does not.
+            // From experience, this code runs in `<sourceCodePath>/bin/output/`, so we store the file two directories above.
+            // We do this because the /bin/output/ directory gets overridden during the build process, which happens automatically
+            // when `func host start` is re-invoked.
+            string evidenceFile = System.IO.Path.Combine(System.IO.Directory.GetCurrentDirectory(), "..", "..", "replayEvidence");
+            bool isTheFirstReplay = !System.IO.File.Exists(evidenceFile);
+            if (isTheFirstReplay)
+            {
+                System.IO.File.Create(evidenceFile).Close();
+
+                // force the process to run out of memory
+                List<byte[]> data = new List<byte[]>();
+
+                for (int i = 0; i < 10000000; i++)
+                {
+                    data.Add(new byte[1024 * 1024 * 1024]);
+                }
+            }
+            
+            // assuming the orchestrator survived the OOM, delete the evidence file and return
+            System.IO.File.Delete(evidenceFile);
+            return Task.CompletedTask;
+        }
+        
+        [Function(nameof(ProcessExitOrchestrator))]
+        public static Task ProcessExitOrchestrator(
+            [OrchestrationTrigger] TaskOrchestrationContext context)
+        {
+            // this orchestrator is not deterministic, on purpose.
+            // we use the non-determinism to force a sudden process exit on only the first replay
+            
+            // check if a file named "replayEvidence" exists in source code directory, create it if it does not.
+            // From experience, this code runs in `<sourceCodePath>/bin/output/`, so we store the file two directories above.
+            // We do this because the /bin/output/ directory gets overridden during the build process, which happens automatically
+            // when `func host start` is re-invoked.
+            string evidenceFile = System.IO.Path.Combine(System.IO.Directory.GetCurrentDirectory(), "..", "..", "replayEvidence");
+            bool isTheFirstReplay = !System.IO.File.Exists(evidenceFile);
+            if (isTheFirstReplay)
+            {
+                System.IO.File.Create(evidenceFile).Close();
+
+                // simulate sudden crash
+                Environment.FailFast("Simulating crash!");
+            }
+            
+            // assuming the orchestrator survived the OOM, delete the evidence file and return
```

Comment:
Does this comment need to be updated?

Reply:
Good catch, this is the result of copy-pasting.
Updated: https://github.com/Azure/azure-functions-durable-extension/pull/2922/commits/a70d0146fae6771f6cfd834a374407045043a9ec